app.js

```diff
@@ -0,0 +1,44 @@
+const countSentences = (text) => {
+  let myArray = text.split('')
+  myArray = myArray.filter(text => text === '?' || text === '!' || text === '.')
+  return myArray.length
+}
+
+const countWords = (text) => {
+  let trimmed = text.replace(/\s+/g, ' ').trim();
+  let textArray = trimmed.split(' ');
+  let numOfWords = textArray.length;
+  if (textArray[0] == '') {
+    textArray = 0
+  }
+  return numOfWords;
+}
+
+const countLetters = (text) => {
+  let myArray = text.split('')
+  myArray = myArray.filter(text => text !== ' ' && text !== '?' && text !== '!' && text !== '.' && text !== "'")
+  return myArray.length
+}
+
+function handleCounter(event) {
+  event.preventDefault();
+  let text = document.getElementById('textarea').value;
+
+  let selectedCountOption = document.querySelector('input[name="countOption"]:checked').value;
+  let count = 0;
+
+  if (selectedCountOption === 'sentences') {
+    count = countSentences(text);
+  } else if (selectedCountOption === 'words') {
+    count = countWords(text);
+  } else if (selectedCountOption === 'letters') {
+    count = countLetters(text);
+  }
+
+  document.getElementById('wordCount').textContent = `The result is ${count}`;
+}
+
+
+
+
+
```

Comment:
Hi, dear Alvard.  I'm glad to have the opportunity to write a review about your code!
Your code works fine. There is only a small problem of counting words and sentences. When the user types a few characters like "aaa ....", it is counted as 2 words and 4 sentences, but there is actually 1 word and 1 sentence. You can fix this by splitting the string and checking if it is empty after splitting or not.